packages/utilities/src/utilities/cloudAccountAuthentication/cloudAccountAuthentication.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Cookies from 'js-cookie';
+
+/**
+ * The cookie name for determining user login status for cloud.ibm.com.
+ *
+ * @type {string}
+ * @private
+ */
+const _cookieName = 'com.ibm.cloud.iam.LoggedIn.manual';
+
+/**
+ * Utility to to determine user login status via cookie or api
+ */
+class cloudAccountAuthentication {
+  /**
+   * retrieve the cloud login status via cookie
+   *
+   * @example
+   * import { cloudAccountAuthentication } from '@carbon/ibmdotcom-utilities';
+   *
+   * const status = cloudAccountAuthentication.checkCookie();
+   *
+   * @returns {string} string determining login status
+   */
+  static checkCookie() {
+    const cloudLogin = Cookies.get(_cookieName);
+
+    return cloudLogin === '1' ? 'authenticated' : 'anonymous';
+  }
+
+  /**
+   * retrieve the cloud login status via api
+   *
+   * @example
+   * import { cloudAccountAuthentication } from '@carbon/ibmdotcom-utilities';
+   *
+   * const status = cloudAccountAuthentication.checkAPI();
+   *
+   * @returns {string} string determining login status
+   */
+  static async checkAPI() {
+    const cloudLogin = await fetch(
+      'https://cloud.ibm.com/api/v6/selected-account?profile=true',
```

Comment:
Seems like this is hard coded to call production, we have other environments like test.cloud.ibm.com and dev.test.console.cloud.ibm.com. This should be a relative url to make sure those envs work as well.